ramalama/cli.py

```diff
@@ -256,6 +257,17 @@ def version_parser(subparsers):
     parser.set_defaults(func=version_cli)
 
 
+def rm_parser(subparsers):
+    parser = subparsers.add_parser("rm", help="Remove AI model from local storage")
+    parser.add_argument("model", help="Name of the model to remove")
+    parser.set_defaults(func=rm_cli)
+
+
+def rm_cli(args):
+    model = New(args.model)
```

Comment:
Capitalize model

Reply:
There's a lot of similar references, do I capitalise everywhere? :

```
ramalama/cli.py
80:    model = New(str(transport))
97:    model = New(str(transport))
210:    model = New(args.model)
226:    model = New(args.model)
239:    model = New(args.model)
251:    model = New(args.model)
267:    model = New(args.model)

ramalama/model.py
9:    model = ""
15:        self.model = model
```

Reply:
I think I see what you mean now, I'll try to be consistent with your recent PR

Reply:
Done